### ISSUE TITLE
Reference is removing umlauts

### ISSUE THREAD

**Issue by BoneGoat** [NVIDIA/NeMo#730]
I'm training QuartzNet15x5 on my own training data and my own vocab. When the log prints the reference it replaces å ä and ö with a a and o:

```
[NeMo I 2020-06-14 09:25:14 helpers:73] Reference: sa jag hoppade in i restaurangkoket och hjalpte till att gora sillinlaggningar som jag ar mycket fascinerad av
```
From train.json:
```
{"audio_filepath": "/path/to/wav", "duration": 5.344, "text": "så jag hoppade in i restaurangköket och hjälpte till att göra sillinläggningar som jag är mycket fascinerad av"}
```

Very early on in the training the model predicts:
```
[NeMo I 2020-06-14 09:18:35 helpers:72] Prediction: köfvkåuxdåixdfmkåakfvkn
```

But when running inference it never predicts å ä or ö. Is my setup broken?

yaml:
```
header: {collection_type: asr, collection_version: null, full_spec: nemo.collections.asr.models.asrconvctcmodel.QuartzNet,
  nemo_core_version: 0.11.0b6}
init_params:
  decoder_params:
    header: {full_spec: nemo.collections.asr.JasperDecoderForCTC}
    init_params:
      feat_in: 1024
      num_classes: 33
      vocabulary: [" ", "a", "b", "c", "d", "e", "é", "è", "f", "g", "h", "i", "j", "k", "l", "m", "n", "o", "p", "q", "r", "s", "t", "u", "ü", "v", "w", "x", "y", "z", "å", "ä", "ö"]
  encoder_params:
    header: {full_spec: nemo.collections.asr.JasperEncoder}
    init_params:
      activation: relu
      conv_mask: true
      feat_in: 64
      jasper:
      - dilation: [1]
        dropout: 0.0
        filters: 256
        kernel: [33]
        repeat: 1
        residual: false
        separable: true
        stride: [2]
      - dilation: [1]
        dropout: 0.0
        filters: 256
        kernel: [33]
        repeat: 5
        residual: true
        separable: true
        stride: [1]
      - dilation: [1]
        dropout: 0.0
        filters: 256
        kernel: [33]
        repeat: 5
        residual: true
        separable: true
        stride: [1]
      - dilation: [1]
        dropout: 0.0
        filters: 256
        kernel: [33]
        repeat: 5
        residual: true
        separable: true
        stride: [1]
      - dilation: [1]
        dropout: 0.0
        filters: 256
        kernel: [39]
        repeat: 5
        residual: true
        separable: true
        stride: [1]
      - dilation: [1]
        dropout: 0.0
        filters: 256
        kernel: [39]
        repeat: 5
        residual: true
        separable: true
        stride: [1]
      - dilation: [1]
        dropout: 0.0
        filters: 256
        kernel: [39]
        repeat: 5
        residual: true
        separable: true
        stride: [1]
      - dilation: [1]
        dropout: 0.0
        filters: 512
        kernel: [51]
        repeat: 5
        residual: true
        separable: true
        stride: [1]
      - dilation: [1]
        dropout: 0.0
        filters: 512
        kernel: [51]
        repeat: 5
        residual: true
        separable: true
        stride: [1]
      - dilation: [1]
        dropout: 0.0
        filters: 512
        kernel: [51]
        repeat: 5
        residual: true
        separable: true
        stride: [1]
      - dilation: [1]
        dropout: 0.0
        filters: 512
        kernel: [63]
        repeat: 5
        residual: true
        separable: true
        stride: [1]
      - dilation: [1]
        dropout: 0.0
        filters: 512
        kernel: [63]
        repeat: 5
        residual: true
        separable: true
        stride: [1]
      - dilation: [1]
        dropout: 0.0
        filters: 512
        kernel: [63]
        repeat: 5
        residual: true
        separable: true
        stride: [1]
      - dilation: [1]
        dropout: 0.0
        filters: 512
        kernel: [75]
        repeat: 5
        residual: true
        separable: true
        stride: [1]
      - dilation: [1]
        dropout: 0.0
        filters: 512
        kernel: [75]
        repeat: 5
        residual: true
        separable: true
        stride: [1]
      - dilation: [1]
        dropout: 0.0
        filters: 512
        kernel: [75]
        repeat: 5
        residual: true
        separable: true
        stride: [1]
      - dilation: [2]
        dropout: 0.0
        filters: 512
        kernel: [87]
        repeat: 1
        residual: false
        separable: true
        stride: [1]
      - dilation: [1]
        dropout: 0.0
        filters: 1024
        kernel: [1]
        repeat: 1
        residual: false
        stride: [1]
  preprocessor_params:
    header: {full_spec: nemo.collections.asr.AudioToMelSpectrogramPreprocessor}
    init_params: {dither: 1e-05, features: 64, n_fft: 512, normalize: per_feature,
      pad_to: 16, stft_conv: true, window: hann, window_size: 0.02, window_stride: 0.01}
  spec_augment_params:
    header: {full_spec: nemo.collections.asr.SpectrogramAugmentation}
    init_params: {rect_freq: 50, rect_masks: 5, rect_time: 120}
```

Full log of a training attempt:
```
[NeMo W 2020-06-14 16:13:31 audio_preprocessing:56] Could not import torchaudio. Some features might not work.
[NeMo I 2020-06-14 16:13:31 train-scratch:75] Speech2Text: Will train from scratch using config from quartznet15x5-sv-new.yaml
[NeMo I 2020-06-14 16:13:31 neural_modules:343] Loading configuration of a new Neural Module from the `quartznet15x5-sv-new.yaml` file
[NeMo I 2020-06-14 16:13:31 features:144] PADDING: 16
[NeMo I 2020-06-14 16:13:31 features:152] STFT using conv
[NeMo I 2020-06-14 16:13:32 neural_modules:443] Instantiated a new Neural Module named `audiotomelspectrogrampreprocessor0` of type `AudioToMelSpectrogramPreprocessor`
[NeMo I 2020-06-14 16:13:33 neural_modules:443] Instantiated a new Neural Module named `jasperencoder0` of type `JasperEncoder`
[NeMo I 2020-06-14 16:13:33 neural_modules:443] Instantiated a new Neural Module named `jasperdecoderforctc0` of type `JasperDecoderForCTC`
[NeMo I 2020-06-14 16:13:33 neural_modules:443] Instantiated a new Neural Module named `spectrogramaugmentation0` of type `SpectrogramAugmentation`
[NeMo I 2020-06-14 16:13:33 neural_modules:443] Instantiated a new Neural Module named `quartznet0` of type `QuartzNet`
[NeMo I 2020-06-14 16:13:33 train-scratch:82]


[NeMo I 2020-06-14 16:13:33 train-scratch:83] Speech2Text: Training on 1 GPUs.
[NeMo I 2020-06-14 16:13:33 train-scratch:84] Training <class 'nemo.collections.asr.models.asrconvctcmodel.QuartzNet'> model.
[NeMo I 2020-06-14 16:13:33 train-scratch:85] Training CTC model with alphabet [' ', 'a', 'b', 'c', 'd', 'e', 'é', 'è', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'ü', 'v', 'w', 'x', 'y', 'z', 'å', 'ä', 'ö'].
[NeMo I 2020-06-14 16:13:33 train-scratch:86] Training CTC model with 18929506 weights.


Opening: all-datasets-train-ds.json
[NeMo I 2020-06-14 16:14:46 collections:160] Dataset loaded with 381732 files totalling 370.89 hours
[NeMo I 2020-06-14 16:14:46 collections:161] 490 files were filtered totalling 2.68 hours
[NeMo W 2020-06-14 16:14:46 deprecated:68] Function ``__init__`` is deprecated. It is going to be removed in the 0.12 version. The callback section of NeMo has been updated.
[NeMo I 2020-06-14 16:14:46 train-scratch:130] Will perform evaluation every 1000 steps.
Opening: all-datasets-dev-ds.json
[NeMo I 2020-06-14 16:14:55 collections:160] Dataset loaded with 42470 files totalling 41.40 hours
[NeMo I 2020-06-14 16:14:55 collections:161] 0 files were filtered totalling 0.00 hours
Selected optimization level O1:  Insert automatic casts around Pytorch functions and Tensor methods.

Defaults for this optimization level are:
enabled                : True
opt_level              : O1
cast_model_type        : None
patch_torch_functions  : True
keep_batchnorm_fp32    : None
master_weights         : None
loss_scale             : dynamic
Processing user overrides (additional kwargs that are not None)...
After processing overrides, optimization options are:
enabled                : True
opt_level              : O1
cast_model_type        : None
patch_torch_functions  : True
keep_batchnorm_fp32    : None
master_weights         : None
loss_scale             : dynamic
[NeMo I 2020-06-14 16:14:55 deprecated_callbacks:195] Starting .....
[NeMo I 2020-06-14 16:14:55 callbacks:520] Found 3 modules with weights:
[NeMo I 2020-06-14 16:14:55 callbacks:522] JasperEncoder
[NeMo I 2020-06-14 16:14:55 callbacks:522] JasperDecoderForCTC
[NeMo I 2020-06-14 16:14:55 callbacks:522] QuartzNet
[NeMo I 2020-06-14 16:14:55 callbacks:523] Total model parameters: 37859012
[NeMo I 2020-06-14 16:14:55 callbacks:473] Found checkpoint folder checkpoint. Will attempt to restore checkpoints from it.
[NeMo W 2020-06-14 16:14:55 callbacks:490] For module JasperEncoder, no file matches  in checkpoint
[NeMo W 2020-06-14 16:14:55 callbacks:492] Checkpoint folder checkpoint was present but nothing was restored. Continuing training from random initialization.
[NeMo W 2020-06-14 16:14:55 deprecated:68] Function ``epoch_num`` is deprecated. It is going to be removed in the 0.12 version. epoch_num has been deprecated in favour of epoch.
[NeMo I 2020-06-14 16:14:55 deprecated_callbacks:207] Starting epoch 0
Gradient overflow.  Skipping step, loss scaler 0 reducing loss scale to 32768.0
[NeMo I 2020-06-14 16:14:56 deprecated_callbacks:232] Step: 0
[NeMo I 2020-06-14 16:14:56 helpers:70] Loss: 372.4876708984375
[NeMo I 2020-06-14 16:14:56 helpers:71] training_batch_WER:  100.00%
[NeMo I 2020-06-14 16:14:56 helpers:72] Prediction: järåtjrvålrurstårhrlrjzhxjåjhrjlrjrjrtäyjlkréshfjksäkhjslrlrlthåjrtrtjhctzcvrmiöbtyktklümrhjhjkhtlülraträjlklyfjlamlyhlthjfhcahjmè ljtxtjlhchjrhlhcyjlhjftjarld
[NeMo I 2020-06-14 16:14:56 helpers:73] Reference: hon syns ofta pa piren spanande ut mot branningarna
[NeMo I 2020-06-14 16:14:56 deprecated_callbacks:247] Step time: 1.6250379085540771 seconds
[NeMo I 2020-06-14 16:14:56 deprecated_callbacks:316] Doing Evaluation ..............................
```
And the line from train.json:
```
{"audio_filepath": "/path/to/wav", "duration": 3.52, "text": "hon syns ofta på piren spanande ut mot bränningarna"}
```

**Comment by okuchaiev** [comment]
Hi, it looks like you are trying to train a non-En model. Please set "normalize_transcripts=False" when you are instantiating a AudioToTextDataLayer https://nvidia.github.io/NeMo/collections/nemo_asr.html?highlight=audiototextdatalayer#nemo.collections.asr.data_layer.AudioToTextDataLayer

**Comment by BoneGoat** [comment]
Thank you for your help!
